src/status_im/ui/screens/desktop/main/chat/styles.cljs

```diff
@@ -37,9 +37,13 @@
    :width              230})
 
 (def author
-  {:color         colors/black
-   :font-size     14
-   :font-weight   :bold})
+  {:color       colors/black
+   :font-weight 600
```

Comment:
font-weight has to be a string

Reply:
good catch, interesting why there is a difference between 600 and 500 then

Reply:
Maybe it's only a problem on Linux

Reply:
With font weight 400 is regular 500 is medium, 600 semi bold and 700 bold :)